concordium_p2p_rpc.proto

```diff
@@ -4,223 +4,433 @@ import "google/protobuf/wrappers.proto";
-  //! Peer client software version
+
+  // Node software version.
   rpc PeerVersion(Empty) returns (StringResponse) {}
-  //! Stats for connected peers
+
+  // Stats for connected peers.
   rpc PeerStats(PeersRequest) returns (PeerStatsResponse) {}
-  //! List of connected peers
+
+  // List of connected peers.
   rpc PeerList(PeersRequest) returns (PeerListResponse) {}
+
+  // Ban a the given peer.
   rpc BanNode(PeerElement) returns (BoolResponse) {}
+
+  // Unban the given peer.
   rpc UnbanNode(PeerElement) returns (BoolResponse) {}
+
+  // Join the provided network.
   rpc JoinNetwork(NetworkChangeRequest) returns (BoolResponse) {}
+
+  // Leave the provided network.
   rpc LeaveNetwork(NetworkChangeRequest) returns (BoolResponse) {}
-  //! Get information about the running Node
+
+  // Get information about the running node.
   rpc NodeInfo(Empty) returns (NodeInfoResponse) {}
-  //! see https://gitlab.com/Concordium/notes-wiki/wikis/Consensus-queries#getconsensusstatus
+
+  // Get information about the consensus.
+  // A JSON schema for the return type is provided at: https://developer.concordium.software/en/mainnet/net/references/grpc.html.
   rpc GetConsensusStatus(Empty) returns (JsonResponse) {}
-  //! see https://gitlab.com/Concordium/notes-wiki/wikis/Consensus-queries#getblockinfo
+
+  // Get information about the block.
+  // A JSON schema for the return type is provided at: https://developer.concordium.software/en/mainnet/net/references/grpc.html.
   rpc GetBlockInfo(BlockHash) returns (JsonResponse) {}
-  //! see https://gitlab.com/Concordium/notes-wiki/wikis/Consensus-queries#getancestors
+
+  // Get ancestors for the provided block.
+  // A JSON schema for the return type is provided at: https://developer.concordium.software/en/mainnet/net/references/grpc.html.
   rpc GetAncestors(BlockHashAndAmount) returns (JsonResponse) {}
-  //! see https://gitlab.com/Concordium/notes-wiki/wikis/Consensus-queries#getbranches
+
+  // Get the current branches.
+  // A JSON schema for the return type is provided at: https://developer.concordium.software/en/mainnet/net/references/grpc.html.
   rpc GetBranches(Empty) returns (JsonResponse) {}
-  //! Get the blocks at the given height
+
+  // Get the blocks at the given height.
+  // A JSON schema for the return type is provided at: https://developer.concordium.software/en/mainnet/net/references/grpc.html.
   rpc GetBlocksAtHeight(BlockHeight) returns (JsonResponse) {}
-  //! Submit a local transaction
+
+  // Submit a transaction.
   rpc SendTransaction(SendTransactionRequest) returns (BoolResponse) {}
 
-  //! Start the baker in the consensus module
+  // Start the baker in the consensus module.
   rpc StartBaker(Empty) returns (BoolResponse) {}
-  //! Stop the baker in the consensus module
+
+  // Stop the baker in the consensus module.
   rpc StopBaker(Empty) returns (BoolResponse) {}
 
+  // Get a list of accounts that existed when the given block was created.
```

Comment:
The phrasing "that existed when the given block was created" here and below is a bit too imprecise.

Something like 
```
... that exist in the state after the given block
```
or 
```
... that exist in the state at the end of the given block
```
would be more accurate.